components/ClusterDisplayProvider.vue

```diff
@@ -42,8 +42,13 @@ export default {
             const nodeTemplateId = firstNodePool?.spec?.nodeTemplateName;
             const normalizedId = nodeTemplateId.split(':').join('/');
             const nodeTemplate = this.nodeTemplates.find(nt => nt.id === normalizedId);
+            const nodeDriver = nodeTemplate?.spec?.driver || null;
+
+            if (nodeDriver) {
+              if (this.$store.getters['i18n/exists'](`cluster.nodeDriver.displayName.${ nodeDriver.toLowerCase() }`)) {
+                return this.$store.getters['i18n/t'](`cluster.nodeDriver.displayName.${ nodeDriver.toLowerCase() }`);
+              }
 
-            if (nodeTemplate?.spec?.driver) {
               return capitalize( nodeTemplate.spec.driver );
```

Comment:
The dropped if here means this will blow up if the nodeTemplate isn't found for some reason

Reply:
I added an additional check and if that fails we will return the custom short label again.